app/src/app/[lng]/[inventory]/TabHeader.tsx

```diff
@@ -0,0 +1,81 @@
+import { TFunction } from "i18next";
+import { Badge, Box, Heading, HStack, Link, Text } from "@chakra-ui/react";
+import { Trans } from "react-i18next/TransWithoutContext";
+import { MdOutlineAccountTree, MdOutlineCalendarToday } from "react-icons/md";
+
+export function TabHeader({
+  t,
+  title,
+  year,
+}: {
+  t: TFunction<string, undefined>;
+  year: number | undefined;
+  title: string;
+}) {
+  return (
+    <>
+      <Box className="flex items-center gap-3">
+        <Heading fontSize="headline.sm" fontWeight="semibold" lineHeight="32">
+          <Trans t={t}>{title}</Trans>
+        </Heading>
+      </Box>
+      <Box className="flex items-center gap-3">
+        <Badge
+          borderWidth="1px"
+          borderColor="border.neutral"
+          py="4px"
+          px="8px"
```

Comment:
This works too: `py={1} px={2}`. Definitions for what these numbers mean are here in the ChakraUI docs: https://v2.chakra-ui.com/docs/styled-system/theme#spacing